ci/docker/riscv64gc-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        gcc libc6-dev qemu-user ca-certificates \
+        gcc-riscv64-linux-gnu libc6-dev-riscv64-cross \
+        qemu-system-riscv64 linux-headers-generic
+
+ENV CARGO_TARGET_RISCV64GC_UNKNOWN_LINUX_GNU_LINKER=riscv64-linux-gnu-gcc \
+    CARGO_TARGET_RISCV64GC_UNKNOWN_LINUX_GNU_RUNNER="qemu-riscv64 -L /usr/riscv64-linux-gnu" \
```

Comment:
Sometimes I've seen rust test suites hang on risc-v while using qemu user emulation, but the same tests work correctly under full system emulation. If it works okay here then great, but it is something to watch out for.

I believe this effects other targets too as it is mentioned in the [cross readme](https://github.com/rust-embedded/cross/blob/master/README.md) (just above the supported targets table).

Reply:
Hm, I have never seen that failure on other targets enabled here. Let's see whether it works well for a while.